rmw_zenoh_cpp/src/detail/rmw_subscription_data.cpp

```diff
@@ -224,7 +225,7 @@ bool SubscriptionData::init()
 
         sub_data->add_new_message(
           std::make_unique<SubscriptionData::Message>(
-            sample.get_payload().clone(),
+            sample.get_payload().as_vector(),
```

Comment:
Hm, I have a serious concern about this, but I'll leave a comment in the other PR since it is already merged.

Reply:
Sure. Also answered in that PR :smile:.